apps/src/sites/studio/pages/levels-lab2-main.js

```diff
@@ -0,0 +1,15 @@
+import $ from 'jquery';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Provider} from 'react-redux';
+import {getStore} from '@cdo/apps/redux';
+import Lab2 from '@cdo/apps/labs/lab2';
+
+$(document).ready(function () {
+  ReactDOM.render(
+    <Provider store={getStore()}>
```

Comment:
Can we move the `<Provider>` wrapper into `<Lab2>` as well?

Reply:
Ah, we apparently need a provider outside of `Lab2` if we want to be able to use a selector in `Lab2`.

Reply:
Ah right. could we still do something like
```
const Lab2WithProvider = <Provider><Lab2/></Provider>
export default Lab2WithProvider
```
just so future importers of Lab2 don't also need to worry about also wrapping it in a `Provider`?

Reply:
Good idea.  Got it working with a slight variation.